<launch>
	<node name="colorblob" pkg="borg_nao" type="colorblob_node" output="screen" >
		<param name="image_topic" value="/NAO/image_bottom" />
		<param name="visual_mode" value="true" />
		
		<param name="red_hue_upper" value="255" />
		<param name="red_hue_lower" value="114" />
		<param name="red_saturation_upper" value="255" />
		<param name="red_saturation_lower" value="129" />
		<param name="red_value_upper" value="255" />
		<param name="red_value_lower" value="69"/>
		
		<param name="yellow_hue_upper" value="255" />
		<param name="yellow_hue_lower" value="85" />
		<param name="yellow_saturation_upper" value="255" />
		<param name="yellow_saturation_lower" value="152" />
		<param name="yellow_value_upper" value="255" />
		<param name="yellow_value_lower" value="66"/>


		<param name="blue_hue_upper" value="255" />
		<param name="blue_hue_lower" value="0" />
		<param name="blue_saturation_upper" value="255" />
		<param name="blue_saturation_lower" value="0" />
		<param name="blue_value_upper" value="255" />
		<param name="blue_value_lower" value="0"/>
		
		<param name="def_hue_upper" value="0" />
		<param name="def_hue_lower" value="0" />
		<param name="def_saturation_upper" value="0" />
		<param name="def_saturation_lower" value="0" />
		<param name="def_value_upper" value="0" />
		<param name="def_value_lower" value="0"/>
	</node>
</launch>


